sockeye/inference.py

```diff
@@ -353,6 +364,16 @@ def length_ratio_mean(self) -> float:
     def length_ratio_std(self) -> float:
         return self.config.config_data.data_statistics.length_ratio_std
 
+def load_source_factor_vocabs(model_folder: str, num_factors: Optional[int] = 0) -> List[Dict[str,int]]:
+    """
+    Loads source factor vocabs of the form "vocab.src.N" until one can no longer be found.
+    This could alternately be done by first reading the number of factors from the config file.
+
+    :param model_folder: The location of the model folder
+    :return: The list of source factor vocabs found in the model folder.
+    """
+    return [vocab.vocab_from_json_or_pickle(os.path.join(model_folder, C.VOCAB_SRC_NAME) + "." + str(fi)) for fi in range(num_factors)]
```

Comment:
Not that important but I'd prefer string formatting over concatenation :)

Reply:
agreed, done